platform/admin/core/admin-core-migration-commands/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -42,21 +42,8 @@
     <bean id="security" class="org.codice.ddf.security.common.Security"
           factory-method="getInstance"/>
 
-    <!-- Base URI for DDF home -->
-    <bean id="ddfHomeFile" class="java.io.File">
-        <argument value="${ddf.home}"/>
-    </bean>
-
-    <bean id="ddfHomeUri" class="java.net.URI" factory-ref="ddfHomeFile" factory-method="toURI"/>
-
-    <!-- Default Directory Export Path -->
-    <bean id="defaultExportDirectoryUri" class="java.nio.file.Uri" factory-ref="ddfHomeUri"
-          factory-method="resolve">
+    <bean id="defaultExportDirectoryPath" class="java.io.File">
         <argument value="${ddf.home}/etc/exported"/>
```

Comment:
Investigate passing in the root as a single argument value and then passing in the sub-path elements as a list:
```xml
<argument value="${ddf.home}"/>
<argument>
    <list value-type="java.lang.String">
         <value>etc</value>
         <value>exported</value>
    </list>
</argument>
```
Then do something like
```java
Paths.get(arg1, arg2.toArray(new String[arg2.size()]));
```